src/ralph/lib/table/table.py

```diff
@@ -24,10 +24,10 @@ class Table(object):
     Generating contents for table based on predefined columns and queryset.
 
     Example:
-        >>> table = Table(queryset, ['id', 'name'])
+        >>> table = Table(queryset, ['id', ('name', 'My field name')])
```

Comment:
Could you mention about `transpose` option?
What's the meaning of tuple `('name', 'My field name')`?